<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeSetting</class>
 <widget class="QWidget" name="DateTimeSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>155</y>
     <width>190</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(65, 65, 65, 200);
border-radius:10px;
color: rgb(255, 255, 255);
padding:2px;</string>
   </property>
   <property name="text">
    <string>CURRENT TIME(hh:mm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>205</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>105</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>39</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>39</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>39</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(11, 111, 171, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DATE/TIME SETTING</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>55</y>
     <width>190</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(65, 65, 65, 200);
border-radius:10px;
color: rgb(255, 255, 255);
padding:2px;</string>
   </property>
   <property name="text">
    <string>CURRENT DATE(dd/mm/yy):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>242</y>
     <width>210</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>201</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 93, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>273</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label_2</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>title</zorder>
  <zorder>label</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
